### ISSUE TITLE
crash with orientation

### ISSUE THREAD

**Issue by Xplodin** [refinedmods/refinedstorage#3383]
### Describe the bug

placing a cable on a controller caused a server restart / crash loop

### How can we reproduce this bug or crash?

see above

### What Minecraft version is this happening on?

Minecraft 1.18.2

### What Forge version is this happening on?

40.1.80

### What Refined Storage version is this happening on?

1.10.3

### Relevant log output

```shell
https://gist.github.com/Xplodin/fbeea97f11aa9ac6678ba0811695feac
```

**Comment by revilo2218** [comment]
Same thing happened to me, it sucks

**Comment by raoulvdberge** [comment]
Closed in favor of epic issue #3424.
